### ISSUE TITLE
cacheDuration config not working with a string duration interval

### ISSUE THREAD

**Issue by TraffordFewster** [putyourlightson/craft-blitz#536]
We have had some complaints of our site caches not expiring, after digging into it a bit I found that us using a string duration interval is making the cache not have an end date.

### Basic `blitz.php` config file
```
<?php

return [
    '*' => [
        // The amount of time after which the cache should expire (if not 0). See [[ConfigHelper::durationInSeconds()]] for a list of supported value types.
        'cacheDuration' => 'PT1S',
    ]
];
```

With the cache duration set to `'PT1S'`which should be 1 second the cache never expires, if you change it to just a `1` the cache will expire. According to the [ConfigHelper](https://docs.craftcms.com/api/v4/craft-helpers-confighelper.html#public-methods) docs a string [duration interval](https://en.wikipedia.org/wiki/ISO_8601#Durations) should be accepted but it doesn't appear to work. 

### Versions
- `craftcms/cms` 4.4.15
- `putyourlightson/craft-blitz` 4.4.6
- PHP 8.1.19
- MySQL 5.7.23
- Redis 7.0.0

**Comment by bencroker** [comment]
Thanks for reporting this! Fixed in https://github.com/putyourlightson/craft-blitz/commit/a487ca51985810c93678cbd51e65979cc26656c7 for the next release.

**Comment by bencroker** [comment]
Released in version [4.5.0](https://github.com/putyourlightson/craft-blitz/releases/tag/4.5.0).
